src/fidasim.f90

```diff
@@ -9503,11 +9503,11 @@ subroutine doppler_stark(vecp, vi, fields, lambda0, lambda)
         !+ Reference wavelength [nm]
     real(Float64), dimension(n_stark), intent(out) :: lambda
         !+ Wavelengths [nm]
-    real(Float64), dimension(15) :: wavel
```

Comment:
Oh setting 15 instead of n_stark must be the issue, not the way the array I was setting up the arrays. Will change in next commit